core/templates/dev/head/pages/exploration_player/correctness_footer_directive.html

```diff
@@ -20,6 +20,9 @@
     text-align: center;
   }
 
+  correctness-footer .inner-container-alignment {
+    padding-left: 60px;
+  }
```

Comment:
@bansalnitish, FYI the hardcoded pixel value here won't work for varying screen sizes specially on mobiles. You can instead use `text-align: center` or some other attributes that does not require pixel values.

Reply:
I think adding padding-left: 60px is fine. I need to write a media query to remove this padding when size exceeds certain limit. The footer was originally exactly at the center of the page but the tutor card is not in center because it has images of oppia logo with it. You can have a look at this code snippet in tutor_card_directive.html.
```
.conversation-skin-tutor-card,
  .conversation-skin-future-tutor-card {
    border-radius: 2px;
    padding-left: 87px;
    text-align: left;
  }
```
Here also padding-left is hardcoded.
I am looking for a fix and will update the PR as soon as possible. Do let me know if I'm wrong. Thanks!